client/webserver/locales/en-us.go

```diff
@@ -158,12 +158,14 @@ var EnUS = map[string]string{
 	"seed_implore_msg":            "You should carefully write down your application seed and save a copy. Should you lose access to this machine or the critical application files, the seed can be used to restore your DEX accounts and native wallets. Some older accounts cannot be restored from seed, and whether old or new, it's good practice to backup the account keys separately from the seed.",
```

Comment:
> Some older accounts cannot be restored from seed, and whether old or new, it's good practice to backup the account keys separately from the seed.

I wonder whether this part is still relevant ? Or maybe this warning has already served its purpose and everyone migrated their accounts ?

And, I'm not sure what `account keys` mentioned here are (is it also an old artifact new users shouldn't worry about ?)

my main concern is - the simpler (and shorter) the text is, the better UX is for the user

Reply:
I don't think the message ever made much sense.  The seed gives your account keys, which are is the key pair that uniquely identifies you to the server.  The seed is all anyone needs, so why is is good practice to backup/export every DEX account?  I don't think it is.

Reply:
okay, I've slightly updated texts for `seed_implore_msg` and `pw_for_seed` to address that,

let me know if you want different phrasing for these